cmd/insights/repositories.go

```diff
@@ -1,36 +1,35 @@
 package insights
 
 import (
-	"context"
 	"errors"
 	"fmt"
-	"slices"
 	"strconv"
 	"strings"
 	"sync"
 
 	bubblesTable "github.com/charmbracelet/bubbles/table"
-	"github.com/open-sauced/go-api/client"
 	"github.com/spf13/cobra"
 
-	"github.com/open-sauced/pizza-cli/pkg/api"
+	"github.com/open-sauced/pizza-cli/api"
+	"github.com/open-sauced/pizza-cli/api/services/contributors"
+	"github.com/open-sauced/pizza-cli/api/services/histogram"
 	"github.com/open-sauced/pizza-cli/pkg/constants"
 	"github.com/open-sauced/pizza-cli/pkg/utils"
 )
 
 type repositoriesOptions struct {
 	// APIClient is the http client for making calls to the open-sauced api
-	APIClient *client.APIClient
+	APIClient *api.Client
 
 	// Repos is the array of git repository urls
 	Repos []string
 
 	// FilePath is the path to yaml file containing an array of git repository urls
 	FilePath string
 
-	// Period is the number of days, used for query filtering
+	// RangeVal is the number of days, used for query filtering
```

Comment:
Is `Range` reserved or is `SomeNameVal` the naming convention for certain things in Go?

Reply:
Yes, "range" is reserved. It's a keyword for iterating slices / maps / etc.:

```go
    sum := 0
    for _, num := range []int{2, 3, 4} {
        sum += num
    }
    fmt.Println("sum:", sum)
```

https://go.dev/play/p/_wuEx2nR4VS

https://gobyexample.com/range